### ISSUE TITLE
Crash on QMServiceManager Init

### ISSUE THREAD

**Issue by Yostra** [QuickBlox/q-municate-services-ios#165]
QMSM was added manually follwoing instruction at: https://github.com/QuickBlox/q-municate-services-ios
crash happens when this line tries to get executed: 

```
[ServicesManager.instance.currentUser login];
```

Tried building bundle QMUserCacheModel and adding it into project, but that didn't fix issue

```
+ (void)setupDBWithStoreNamed:(NSString *)storeName
{
    NSManagedObjectModel *model = [NSManagedObjectModel QM_newModelNamed:@"QMUsersModel.momd"
                             inBundleNamed:@"QMUsersCacheModel.bundle"];
    _usersCacheInstance = [[QMUsersCache alloc] initWithStoreNamed:storeName
                                                             model:model
                                                        queueLabel:"com.qmservices.QMUsersCacheQueue"];
}
```

this is where it crashes exactly^

```
 Init - QMAuthService service...
2016-03-27 16:33:43.269 TestyApp[804:14489] Init - QMChatService service...
2016-03-27 16:33:43.281 TestyApp[804:14489] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[NSURL initFileURLWithPath:]: nil string parameter'
*** First throw call stack:
(
    0   CoreFoundation                      0x00000001117d8e65 __exceptionPreprocess + 165
    1   libobjc.A.dylib                     0x0000000110ec8deb objc_exception_throw + 48
    2   CoreFoundation                      0x00000001117d8d9d +[NSException raise:format:] + 205
    3   Foundation                          0x0000000110a719b7 -[NSURL(NSURL) initFileURLWithPath:] + 131
    4   Foundation                          0x0000000110a7191f +[NSURL(NSURL) fileURLWithPath:] + 45
    5   Range                               0x000000010e69b30e +[NSManagedObjectModel(QMCDRecord) QM_newModelNamed:inBundleNamed:] + 270
    6   Range                               0x000000010e649ece +[QMUsersCache setupDBWithStoreNamed:] + 94
    7   Range                               0x000000010e646bc2 -[QMServicesManager init] + 498
    8   Range                               0x000000010e4fd17e -[ServicesManager init] + 62
    9   Range                               0x000000010e646d72 __29+[QMServicesManager instance]_block_invoke + 50
    10  libdispatch.dylib                   0x000000011335e49b _dispatch_client_callout + 8
    11  libdispatch.dylib                   0x0000000113349e28 dispatch_once_f + 543
    12  Range                               0x000000010e646d1b +[QMServicesManager instance] + 139
    13  Range                               0x000000010e4eaefc -[AppDelegate application:didFinishLaunchingWithOptions:] + 732
    14  UIKit                               0x0000000111b7c1f1 -[UIApplication _handleDelegateCallbacksWithOptions:isSuspended:restoreState:] + 272
    15  UIKit                               0x0000000111b7d397 -[UIApplication _callInitializationDelegatesForMainScene:transitionContext:] + 3415
    16  UIKit                               0x0000000111b83cc6 -[UIApplication _runWithMainScene:transitionContext:completion:] + 1760
    17  UIKit                               0x0000000111b80e7b -[UIApplication workspaceDidEndTransaction:] + 188
    18  FrontBoardServices                  0x0000000115039754 -[FBSSerialQueue _performNext] + 192
    19  FrontBoardServices                  0x0000000115039ac2 -[FBSSerialQueue _performNextFromRunLoopSource] + 45
    20  CoreFoundation                      0x0000000111704a31 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
    21  CoreFoundation                      0x00000001116fa95c __CFRunLoopDoSources0 + 556
    22  CoreFoundation                      0x00000001116f9e13 __CFRunLoopRun + 867
    23  CoreFoundation                      0x00000001116f9828 CFRunLoopRunSpecific + 488
    24  UIKit                               0x0000000111b807cd -[UIApplication _run] + 402
    25  UIKit                               0x0000000111b85610 UIApplicationMain + 171
    26  Range                               0x000000010e504dcf main + 111
    27  libdyld.dylib                       0x000000011339292d start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by Yostra** [comment]
Fixed manually by moving: QMUserModel.momd
Before:
![screen shot 2016-03-27 at 9 59 19 pm](https://cloud.githubusercontent.com/assets/7254853/14069623/5f3df306-f46a-11e5-9c6b-2711bf51a362.png)
After:
![screen shot 2016-03-27 at 9 59 55 pm](https://cloud.githubusercontent.com/assets/7254853/14069621/5d521086-f46a-11e5-8854-7fb2150c612f.png)

**Comment by Raikerian** [comment]
Fixed in 0.3.9.
